readme.md

```diff
@@ -23,10 +23,51 @@ console.log(random(), random(), random());
 
 ## API
 
-### uniqueRandom(minimum, maximum)
+### uniqueRandom(minimum, maximum, options?)
 
 Returns a function, that when called, will return a random number that is never the same as the previous.
 
+#### minimum
+
+Type: `number`\
+The minimum number that can be returned.
+
+#### maximum
+
+Type: `number`\
+The maximum number that can be returned.
+
+#### options
+
+Type: `object`
+
+##### exhaustive
+
+Type: `boolean`\
+Default: `false`\
```

Comment:
There should be a newline between the argument type information and default, and the description.